superset-frontend/src/components/ErrorMessage/ErrorMessageWithStackTrace.tsx

```diff
@@ -43,6 +43,7 @@ export default function ErrorMessageWithStackTrace({
   link,
   stackTrace,
   source,
+  optionalFunction,
```

Comment:
Do we want to give this a more descriptive name that provides better context?  something like 'errorMitigationFunction'?  I am assuming this is always used in context of link in an error message?